<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceDialog</class>
 <widget class="QDialog" name="ReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="find_next">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查找下一个</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>查找内容</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="case_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>区分大小写</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="find_content">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cycle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>循环</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="repalce_content">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>替换内容</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replace_one">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>替换</string>
   </property>
  </widget>
  <widget class="QPushButton" name="repalce_all">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>全部替换</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
